decidim-participatory_processes/app/cells/decidim/participatory_process_groups/content_blocks/main_data_cell.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcessGroups
+    module ContentBlocks
+      class MainDataCell < Decidim::ContentBlocks::ParticipatorySpaceMainDataCell
+        include Decidim::SanitizeHelper
+        include Decidim::TwitterSearchHelper
+
+        delegate :title, :description, :hashtag, :group_url, :meta_scope, to: :resource
+
+        private
+
+        def extra_classes
+          "participatory-space-group__content-block"
+        end
+
+        def title_text
+          translated_attribute(title)
+        end
+
+        def description_text
+          decidim_sanitize_editor_admin translated_attribute(description)
+        end
+
+        def hashtag_item
+          return if hashtag_text.blank?
+
+          {
+            icon: "twitter-line",
+            text: link_to("##{hashtag_text}", twitter_hashtag_url(hashtag_text), target: "_blank", rel: "noopener")
+          }
+        end
+
+        def processes_count_item
+          {
+            icon: "grid-line",
+            text: t(
+              "decidim.participatory_process_groups.content_blocks.title.participatory_processes",
+              count: cell("decidim/participatory_process_groups/content_blocks/related_processes", model).total_count
```

Comment:
note to self ... what is the cost of this call ? (Need to check the if that total_count is a method or rendering something )

Reply:
The method takes the query used by the cell and returns the elements count, but doesn't render anything.